include/bluetooth/audio/vcs.h

```diff
@@ -77,10 +80,12 @@ struct bt_vcs_included {
  * clients.
  *
  * @param param     Volume Control Service register parameters.
+ * @param[out] vcs  Pointer to the registered Volume Control Service. This will
+ *                  still be valid if the return value is -EALREADY.
  *
  * @return 0 if success, errno on failure.
  */
-int bt_vcs_register(struct bt_vcs_register_param *param);
+int bt_vcs_register(struct bt_vcs_register_param *param, struct bt_vcs **vcs);
```

Comment:
The rest of the API has the struct bt_vcs first.
OTOH, this one is different anyway, as it is a pointer to a pointer, so an exception anyway.
Probably best as it is.

Reply:
As far as I can tell, putting the [out] pointer last is more common thing in Zephyr